benchmark/scripts/Benchmark_DTrace.in

```diff
@@ -136,7 +137,9 @@ class DTraceBenchmarkDriver(perf_test_driver.BenchmarkDriver):
             results.append(result_3)
             results.append(single_iter)
 
-        return DTraceResult(test_name, int(not foundInstability), results)
+        return DTraceResult(
+            test_name, int(not foundInstability), results, self.csv_output
+        )
```

Comment:
`DTraceResult`'s initializer takes exactly 4 arguments. I don't know why it worked before.